### PR TITLE
Add configuration parameters to IronOS docs

### PR DESCRIPTION
**Pull request by tr4nt0r**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a "Configuration parameters" section in the IronOS integration documentation, detailing the automatic detection of the Bluetooth address.
  
- **Documentation**
	- Reorganized existing content for clarity while maintaining core functionality descriptions.

### CODE REVIEW

File: source/_integrations/iron_os.markdown [hunk 1]
```diff
@@ -40,6 +40,13 @@ Home Assistant will detect nearby IronOS devices. Discovered devices will show u
 
 {% include integrations/config_flow.md %}
 
+## Configuration parameters
+
+{% configuration_basic %}
+"Address":
+  description: "The Bluetooth address of the device. This is automatically detected by Home Assistant, so manual entry is not necessary."
```
Comment (andrewsayre) [inline source/_integrations/iron_os.markdown]: > manual entry is not necessary

Does the config flow allow manual entry? Glancing at the code, it appears to be a list: `vol.In(self._discovered_devices)` which would not allow the user to type anything in--only select an option.

Would a description like this be better/more clear?
> The physical address of the detected Bluetooth device.

Comment (tr4nt0r) [reply source/_integrations/iron_os.markdown]: physical as oposed to virtual address? Not sure if that is the right term. The official term is Bluetooth device address (BD_ADDR).

Comment (tr4nt0r) [reply source/_integrations/iron_os.markdown]: I wanted to point out that this isn't a value the user has to type in, in fact, in the config flow the user won't ever see the address, the label of the select displays only the device name. How do other ble integrations describe the address field in the documentation?

File: source/_integrations/iron_os.markdown [hunk 2]
```diff
@@ -40,6 +40,13 @@ Home Assistant will detect nearby IronOS devices. Discovered devices will show u
 
 {% include integrations/config_flow.md %}
 
+## Configuration parameters
```
Comment (frenck) [inline source/_integrations/iron_os.markdown]:
```suggestion
### Configuration parameters
```

**Review by frenck — APPROVED** [review]
Thanks, @tr4nt0r 👍 

../Frenck

**Pull request merged**